modular_citadel/code/modules/client/preferences.dm

```diff
@@ -16,6 +16,7 @@
 	var/widescreenpref = TRUE
 	var/autostand = TRUE
 	var/auto_ooc = FALSE
+	var/no_tetris_storage = FALSE
```

Comment:
No, move it away from modular_file, I'm still planning on deconstructing that ruin of a folder.